build.sbt

```diff
@@ -7,7 +7,7 @@ val defaultSettings = Defaults.coreDefaultSettings ++ xerial.sbt.Sonatype.sonaty
   organization := "com.github.spinalhdl",
   version      := SpinalVersion.all,
   crossScalaVersions := SpinalVersion.compilers,
-  scalaVersion := SpinalVersion.compilers(0),
+  scalaVersion := SpinalVersion.compilers(1),
```

Comment:
This will be done in https://github.com/SpinalHDL/SpinalHDL/pull/918/commits/4c2aa37fa1a767eec8e7ab0771c1407f95887279